libraries/src/application/Helper.php

```diff
@@ -7,14 +7,22 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
+namespace Joomla\Cms\Application;
+
 defined('JPATH_PLATFORM') or die;
 
+use \JFactory;
+use \JLog;
+use \JFilterOutput;
+use \JInstaller;
+use \JText;
+
 /**
  * Application helper functions
  *
  * @since  1.5
  */
-class JApplicationHelper
+class Helper
```

Comment:
I disagree. Your namespace is Application, you should not add the same identifier in the class immediately below it. This is why the `use` construct has the `as` keyword. If we don't follow that rule we will end up with equally pointless class names like `Acme\Example\Admin\Model\AcmeExampleAdminModelFoo`.